sbc_translation/models/correspondence.py

```diff
@@ -22,22 +33,119 @@ class Correspondence(models.Model):
     ##########################################################################
     #                              ORM METHODS                               #
     ##########################################################################
+
     @api.model
     def create(self, vals):
-        """ Create a message for sending the CommKit only if the preferred
-            language is the same as the letter's language else send the letter
-            on the local translate plaforme.
-            """
-
+        """ Create a message for sending the CommKit after be translated on
+             the local translate plaforme.
+        """
         sponsorship = self.env['recurring.contract'].browse(
             vals['sponsorship_id'])
-        if sponsorship.reading_language.id == vals['original_language_id']:
-            letter = super(Correspondence, self).create(vals)
-        else:
+
+        original_lang_id = self.env['res.lang.compassion'].browse(
```

Comment:
You could start the method with `letter = super(..)` so that you can then user `original_lang = letter.original_language_id` and `sponsorship = letter.sponsorship_id`.